src/server.rs

```diff
@@ -1510,6 +1510,17 @@ impl Default for ServerStats {
     }
 }
 
+macro_rules! set_percentage_stat {
```

Comment:
why a macro and not a function?

Reply:
Picked up code from https://github.com/mozilla/sccache/pull/614 and experienced tunnel vision.

Function is probably a better choice here indeed.